scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,48 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
+description: |
+  Print out the logs for the specified Kafka instance.
+  The following commands will create a directory with all instance information:
+
+  ocm backplane managedjob logs <job-name> > mykafkadebug.tgz
```

Comment:
As all of the script looks correct, making sure that .tgz to the correct file extension

Isn't it .tar.gz?
It would be good to test this before merging

Reply:
Thanks, I've changed that now